menas/src/main/resources/application.properties.template

```diff
@@ -151,7 +151,7 @@ menas.oozie.splineMongoURL=mongodb://localhost:27017
 menas.schemaRegistryBaseUrl=https://localhost:8081
```

Comment:
Discussed with @dk1844 if this should be commented as well (me not sure what is better). Leaving up to him 😉

Reply:
After due consideration, I have commented it out as suggested by @benedeki.